lib/pro/set-status.js

```diff
@@ -13,13 +14,28 @@ function setStatusPro (newStatus, context) {
   const { repository, pull_request: pullRequest } = context.payload
   const { name } = getAppConfig()
 
+  let output = null
+
+  let emojiFound = false
+
+  output = emojiText.convert(newStatus.match, {
+    callback: function (emoji, data) {
+      emojiFound = true
+      return `a ${data.description.toLowerCase()} emoji!`
+    }
+  })
+
+  if (!emojiFound) {
+    output = `"${newStatus.match}"`
+  }
+
   const checkOptions = {
     name: name,
     status: 'in_progress',
     head_branch: '', // workaround for https://github.com/octokit/rest.js/issues/874
     head_sha: pullRequest.head.sha,
     output: {
-      title: 'Work in progress',
+      title: `Title contains ${output}`,
```

Comment:
if you follow my above suggestion, than this needs to be

```suggestion
      title: `Title contains ${newStatus.match}`,
```